### PR TITLE
[ARP] working agreement: make reviewer workflow

### PR DESCRIPTION
**Pull request by ameowlia**
Please leave this PR open until the ARP working group discusses and votes.

This PR is built on top of https://github.com/cloudfoundry/community/pull/155

### CODE REVIEW

File: toc/working-groups/app-runtime-platform/working-agreements/2021-10-28-how-to-make-working-agreements.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# How to Make Changes To Working Agreements
+
+Date of Forum to be Discussed:
```
Comment (ameowlia) [inline toc/working-groups/app-runtime-platform/working-agreements/2021-10-28-how-to-make-working-agreements.md:3]: Nov 3!

File: toc/working-groups/app-runtime-platform/working-agreements/2021-10-29-reviewer-workflows.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+
+## FAQ
+
+❓ **How can I see what is assigned to me?**
+
+Go to the github project.
+Either click on your face on a card that is already assigned to you or
+add the following query param to the URL: `?card_filter_query=assignee:YOUR-GITHUB-USERNAME`
+
```
Comment (geofffranks) [inline toc/working-groups/app-runtime-platform/working-agreements/2021-10-29-reviewer-workflows.md:55]: Can we update this section to also include a link to https://github.com/issues/assigned?q=is%3Aopen+archived%3Afalse+org%3Acloudfoundry to see all the issues/PRs assigned to a person in a single view?

Comment (ameowlia) [reply toc/working-groups/app-runtime-platform/working-agreements/2021-10-29-reviewer-workflows.md:58]: ✅

**Comment by geofffranks** [comment]
+1

**Comment by ctlong** [comment]
+1

**Comment by Gerg** [comment]
+1

**Comment by IvanHristov98** [comment]
+1

**Review by geofffranks — APPROVED** [review]

**Review by dsboulder — APPROVED** [review]
This is very well thought out, thank you so much @ameowlia

**Pull request merged**